bin/node/runtime/pangolin/src/bridges/substrate/millau_messages.rs

```diff
@@ -247,3 +248,17 @@ impl SourceHeaderChain<bp_millau::Balance> for Millau {
 		)
 	}
 }
+
+pub struct MillauCallToPayload;
+
+impl CallToPayload<AccountId, ToMillauMessagePayload> for MillauCallToPayload {
+    fn to_payload(account: AccountId, call: Vec<u8>) -> ToMillauMessagePayload {
+        return FromThisChainMessagePayload::<WithMillauMessageBridge> {
+			spec_version: 1,
+			weight: 100,
```

Comment:
`spec_version` and `weight` should from Target chain(Crab DVM)'s Ethereum transaction input.